src/NavBar.jsx

```diff
@@ -61,8 +61,8 @@ const animationBar = keyframes`
     margin: 0 0 0rem 0;
    }
   100% {
-    height: 1.6rem;
-    margin: 0 0 1rem 0;
+    height: 1.5rem;
+    margin: 0 0 10rem 0;
```

Comment:
can you please keep it as the perviouse code?
``
height: 1.6rem;
 margin: 0 0 1rem
``